### PR TITLE
FOGL-1470 makedeb now checks underlying architecture

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

File: make_deb [hunk 1]
```diff
@@ -22,73 +22,60 @@
...
-              ;;
-          *)
-            echo "Unrecognized option: $i"
-            exit 1
-            ;;
-      esac
-    done
-else
-  echo "See help and select the architecture to use, *x86* or *arm*"
+if [ "$(dpkg --print-architecture)" != "armhf" ]; then
```
Comment (ashish-jabble) [inline make_deb:33]: Is it okay to use? Or with `uname -m`?

Comment (MarkRiddoch) [reply make_deb:33]: I think this is fine as this package only works with ARM because of the GPIO pins

**Review by MarkRiddoch — DISMISSED** [review]

File: make_deb (hunk 1 above)
Comment (praveen-garg) [reply make_deb:33]: I would suggest ` if [ "$(dpkg --print-architecture)" != "arm*" ]; then ` to keep it generic for ARM

**Comment by praveen-garg** [comment]
@ashish-jabble  while you are here, could you please change this line

    new_handle = copy.deepcopy(handle)
to

    new_handle = copy.deepcopy(new_config)

**Comment by ashish-jabble** [comment]
As discussed not required. Hence closed.

**Pull request closed without merging**